### PR TITLE
[13.0][IMP] cb_medical_views: improve patient tree and search

### PR DESCRIPTION
**Pull request by kluna1998**
With this change, we can see the NIF and birthdate of the patients in the tree view and search them by these fields. This pull request is the solution of the issue MR21749.

### CODE REVIEW

File: cb_medical_views/views/medical_patient_views.xml [hunk 1]
```diff
@@ -7,9 +7,21 @@
...
             </search>
         </field>
     </record>
+    <record id="medical_patient_tree" model="ir.ui.view">
+        <field name="name">medical.patient.tree</field>
+        <field name="model">medical.patient</field>
+        <field name="inherit_id" ref="medical_administration.medical_patient_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="vat" string='NIF' />
```
Comment (etobella) [inline cb_medical_views/views/medical_patient_views.xml]: do not translate this way, transaltion should be managed in other ways

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_medical_views/views/medical_patient_views.xml (hunk 1 above)
Comment (kluna1998) [reply cb_medical_views/views/medical_patient_views.xml]: you're right, thx

**Comment by etobella** [comment]
@kluna1998 Can you squash both commits?

**Comment by kluna1998** [comment]
@etobella done 😊

**Pull request merged**